pkg/gui/gui.go

```diff
@@ -311,6 +311,21 @@ func (gui *Gui) resetState(filterPath string, reuseState bool) {
 	if filterPath != "" {
 		screenMode = SCREEN_HALF
 		initialContext = contextTree.LocalCommits
+	} else if gitArgs != "" {
+		switch gitArgs {
+		case "status":
+			screenMode = SCREEN_HALF
```

Comment:
let's move this line just above the switch statement given it's being set to the same value in each leg of the switch statement (and then remove the duplicates). It will be trivially easy to revert this if in future we decide that a particular context needs a different screen mode.